roles/openshift_master/tasks/bootstrap.yml

```diff
@@ -26,3 +26,66 @@
   copy:
     content: "{{ kubeconfig_out.stdout }}"
     dest: "{{ openshift_master_config_dir }}/bootstrap.kubeconfig"
+
+- name: create a temp dir for this work
+  command: mktemp -d /tmp/openshift_node_config-XXXXXX
```

Comment:
We should use somewhere other than /tmp to do this, as /tmp is ugo+r on most systems.  These files looks like they might contain sensitive information.

Reply:
This is a pattern I copied from another role.  I'm open for suggestions.

These files aren't sensitive.